modules/selection/src/main/kotlin/com.r3.corda.lib.tokens.selection/selectors/Selector.kt

```diff
@@ -1,20 +1,18 @@
-package com.r3.corda.lib.tokens.workflows.internal.selection
```

Comment:
DB selection is still in `workflows` shall we move it to this new module?

Reply:
We can, it will make things easier for now.